packages/manager/modules/vps/src/dashboard/tile/configuration/index.html

```diff
@@ -11,6 +11,13 @@
     <oui-tile-definition
         data-term="{{ ::'vps_dashboard_tile_configuration_vcore' | translate }}"
     >
+        <oui-tile-description data-ng-if="$ctrl.statistics">
+            <progress
+                data-ng-class="'oui-progress oui-progress_' + ($ctrl.statistics.cpu <= 50 ? 'success' : $ctrl.statistics.cpu <= 80 ? 'warning' : 'error')"
+                value="{{ $ctrl.statistics.cpu }}"
```

Comment:
Starting to get hard to read 2 nested ternaries in some HTML 
Also: 
 ```suggestion
               class="oui-progress"
                data-ng-class="'oui-progress_' + ($ctrl.statistics.cpu <= 50 ? 'success' : $ctrl.statistics.cpu <= 80 ? 'warning' : 'error')"
                value="{{ $ctrl.statistics.cpu }}"
```